content/about/index.md

```diff
@@ -0,0 +1,122 @@
++++
+fragment = "content"
+weight = 10
+background = "light"
+title = "About Geostyler"
++++
+
+Geostyler is about styling maps. Maybe you are a developer looking for a set of UI components 
+to build custom and flexible styling widgets for your geospatial application. Maybe you 
+are a cartographer trying to get the style you worked so hard on in your favorite (Q)GIS into the 
+web. Or maybe you are a project lead worrying about migrating all your styles to that new fancy 
+rendering engine everyone (and particular your boss) is talking about. The Geostyler ecosystem is 
+here to help. 
+
+###  React Library
+
+One key project of the ecosystem is the React Library providing developers with widgets and 
+tools to build powerfull and flexible UIs to style geodata over the web. To get an idea of 
+it's features and capabilities you can have a look at the [Demo Application](https://geostyler.github.io/geostyler-demo/) or dive into the [Source Code](https://github.com/geostyler/geostyler-demo). If you want to to learn how to build your own custom map styling applications 
+be sure to look into the [tutorials](/#tutorials) and the [documentation](https://geostyler.github.io/geostyler/latest/index.html).
+
+### Style Parsers and Transformations
+
+Geostyler supports multiple style formats used in the open source geospatial ecosystem: 
+QML from QGIS, Mapbox JSON used in MapLibre, OpenLayers Styling Format and the OGC-standardized 
+SLD format used by Geoserver. Each of those can be transformed into one another using 
+internally the Geostyler style format. Each style parser is provided as seperate Github 
+repository and NPM package. You can get an overview of the available style parsers and 
+links [here](/parsers). Be aware that not every styling capability of a particular style 
+is supported. Lucky for you, Geostyler is open source and it is easy to contribute and get 
+involved in the community. Styles can either be transformed using javascript in your application 
+or with the Geostyler client (CLI). 
+
+### Batch Conversion with the Geostyler CLI 
+
+The Geostyler client let's you convert single style files or entire directories from one of the 
+supported styles into another. Assuming you have node installed on your system, installation 
```

Comment:
```
Assuming you have node installed on your system
```

I think this info might be obsolete with the latest changes @geographika did on the geostyler-cli repo :grinning: